optimum/intel/ipex/modeling_base.py

```diff
@@ -226,7 +227,13 @@ def _from_pretrained(
             if is_torch_version("<", "2.1.0"):
                 raise ImportError("`torch>=2.0.0` is needed to trace your model")
 
-            task = cls.export_feature
+            task = TasksManager.infer_task_from_model(model_id, subfolder=subfolder, revision=revision)
```

Comment:
This will not work for local models so I don't think this is something that we should enable